webdriver/tests/get_named_cookie/META.yml

```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1106413
```

Comment:
This bug mentioned the test in a manual import, but isn't about the test but rather something in external/wpt/webmessaging/multi-globals.